egs/wsj/s5/steps/libs/nnet3/train/frame_level_objf/raw_model.py

```diff
@@ -18,7 +18,7 @@
 
 
 def generate_egs_using_targets(data, targets_scp, egs_dir,
-                               left_context, right_context,
+                               left_context, right_context, cmd,
```

Comment:
I don't understand why cmd needs to be provied here if it's available from run_opts.
best to figure out the command at the innermost level rather than changing interfaces.

Reply:
Perhaps I misunderstood your previous comment about the assignment to `run_opts.egs_command`. `args`, which contains `egs_command`, is not being passed to `generate_egs`, only `run_opts` is. That's presumably why `run_opts.egs_command` is assigned to either `args.egs_command` or `args.command` if the former is `None`. 

If you prefer, I can revert the changes introducing the `cmd` parameter and use `command=(run_opts.egs_command if run_opts.egs_command is not None else run_opts.command)` inside `generate_egs` where the command line for `get_egs.sh` is being built.